x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/toolbar/components/alerts_count/alerts_count.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { useEuiTheme } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+const translateUnit = (totalCount: number) =>
+  i18n.translate('xpack.triggersActionsUI.alertsTable.alertsCountUnit', {
+    values: { totalCount },
+    defaultMessage: `{totalCount, plural, =1 {alert} other {alerts}}`,
+  });
+
+export const AlertsCount = ({ count }: { count: number }) => {
+  const { euiTheme } = useEuiTheme();
+
+  const alertCountText = useMemo(
+    () => `${count.toLocaleString()} ${translateUnit(count)}`,
+    [count]
+  );
+
+  return (
+    <span
+      data-test-subj="toolbar-alerts-count"
+      style={{
```

Comment:
can we use the same technique than [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/triggers_actions_ui/public/application/sections/field_browser/field_browser.styles.ts)?
That's the same way than EUI are doing it.

Reply:
https://github.com/elastic/kibana/pull/140473/commits/42790d8327221038d192f4000cc9a69a1e471d1e